packages/flutter_tools/test/web.shard/expression_evaluation_web_test.dart

```diff
@@ -14,215 +14,212 @@ import '../integration.shard/test_driver.dart';
 import '../integration.shard/test_utils.dart';
 import '../src/common.dart';
 
-void batch1() {
-  final BasicProject _project = BasicProject();
-  Directory tempDir;
-  FlutterRunTestDriver _flutter;
-
-  Future<void> initProject() async {
-    tempDir = createResolvedTempDirectorySync('run_expression_eval_test.');
-    await _project.setUpIn(tempDir);
-    _flutter = FlutterRunTestDriver(tempDir);
+void main() {
+  Future<void> failToEvaluateExpression(FlutterTestDriver flutter) async {
```

Comment:
Moved these methods to the top of main.

This PR is easier to look at with whitespace hidden.